### DISCUSSION TITLE
Who's using the Bitcoin Design Guide?

### DISCUSSION THREAD

**Discussion by Bosch-0** [BitcoinDesign/Guide#861] [General]
If you are using the Bitcoin Design Guide in a project or product of yours we'd love to know!

**Comment by crukundo** [comment]
I am currently building EttaWallet: https://github.com/crukundo/EttaWallet to spec with the Bitcoin Design Guide. It's an open source reference bitcoin/lightning wallet. BDC's input would be so important for this.

**Reply by Bosch-0** [reply]
Feel free to sign up for a Bitcoin Crit when you are ready: https://github.com/BitcoinDesign/Meta/issues/322
